lib/alex_koin/coins/coin.ex

```diff
@@ -1,8 +1,9 @@
 defmodule AlexKoin.Coins.Coin do
-  require Ecto.Query
+  import Ecto.Query
 
   use Ecto.Schema
   import Ecto.Changeset
+  alias __MODULE__
```

Comment:
This is really stupid and probably just a pet peeve of mine, but will you group use, import, and alias commands together followed by a space?

Unless for some reason they are order dependent which I don't think they are.